x-pack/legacy/plugins/siem/index.ts

```diff
@@ -23,15 +22,13 @@ import {
   DEFAULT_INTERVAL_VALUE,
   DEFAULT_FROM,
   DEFAULT_TO,
-  DEFAULT_SIGNALS_INDEX,
   ENABLE_NEWS_FEED_SETTING,
   NEWS_FEED_URL_SETTING,
   NEWS_FEED_URL_SETTING_DEFAULT,
-  SIGNALS_INDEX_KEY,
   IP_REPUTATION_LINKS_SETTING,
   IP_REPUTATION_LINKS_SETTING_DEFAULT,
-} from './common/constants';
-import { defaultIndexPattern } from './default_index_pattern';
```

Comment:
It looks like we are not using this elsewhere in the codebase. Do you think we could just delete `default_index_pattern.ts`?

Reply:
That file was meant to be deleted, good catch 👍